pom.xml

```diff
@@ -169,6 +169,21 @@
             <version>4.1.5.RELEASE</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Are you still using Jersey client, if not clean up the dependencies. Also check that the other two dependencies are being used.